app/views/components/_reorderable_list.html.erb

```diff
@@ -0,0 +1,39 @@
+<%
+  items ||= []
+  data_attributes ||= {}
+  data_attributes[:module] = "reorderable-list"
+  order_input_name ||= "reorderable-list-order-#{SecureRandom.hex(4)}"
```

Comment:
Since names don't have to be unique I don't think we really benefit from the random hex here

Reply:
Yep, a leftover. More than that, I think this attribute should be required (no default value provided) so I removed this line.